cmake/linker_script/arm/linker.cmake

```diff
@@ -144,6 +144,7 @@ endif()
 
 include(${COMMON_ZEPHYR_LINKER_DIR}/ram-end.cmake)
 
+zephyr_linker_symbol(SYMBOL __ramfunc_region_start EXPR "ADDR(.ramfunc)")
```

Comment:
@danieldegrasse this line is problematic as it breaks non-ld linkers.
Please see issue created here: https://github.com/zephyrproject-rtos/zephyr/issues/87200